src/doublyLinkedList.js

```diff
@@ -0,0 +1,187 @@
+function Node(data) {
+  this.previous = null
+  this.data = data
+  this.next = null
+}
```

Comment:
You could have used some functions to access the data members. I notice that this is not a class(ES6), but it should be able to work in the same manner.